lms/extensions/feature_flags/_static/feature-flags.css

```diff
@@ -0,0 +1,36 @@
+ul {
+  list-style-type: none;
+  padding: 0;
+}
```

Comment:
These are super-generic classes, so they will inevitably cause conflicts if used outside the feature flag-setting page. That's OK for now though.